pypesto/optimize/ess/sacess.py

```diff
@@ -866,7 +901,76 @@ def dim_refset(x):
             'local_n2': 1,
         },
     ]
+
+    # Set local optimizer
+    for cur_settings in settings:
+        if local_optimizer is True:
+            cur_settings['local_optimizer'] = SacessFidesFactory()
+        elif local_optimizer is not False:
+            cur_settings['local_optimizer'] = local_optimizer
+
     return [
         settings[0],
         *(itertools.islice(itertools.cycle(settings[1:]), num_workers - 1)),
     ]
+
+
+class SacessFidesFactory:
```

Comment:
As far as I can see, the factory is a new addition, providing ess with fitting fides optimisers? Is it a lot more effort to generalise this to more than just fides?